src/styles/stars.scss

```diff
@@ -0,0 +1,19 @@
+.card__stars {
+  display: flex;
+}
+
+.card__stars :not(:last-child) {
+  margin-right: 4px;
+}
+
+.card__stars-star {
```

Comment:
The BEM element should be named 'stars__star' instead of 'card__stars-star' to follow the proper BEM naming conventions.